pkg/event/consumer.go

```diff
@@ -104,8 +99,8 @@ func NewConsumerEventLoop(consumer *kafka.Consumer, handler Eventable) func() {
 					}
 
 					select {
-					case sig := <-sigchan:
-						log.Logger.Info().Msgf("Caught signal %v: terminating\n", sig)
+					case <-ctx.Done():
+						log.Logger.Info().Msgf("Caught signal, terminating consumer\n")
```

Comment:
```suggestion
						log.Logger.Info().Msgf("Context done for NewConsumerEventLoop")
```
I suggest to update the message to indicate the source is the context.